### PR TITLE
BaSP-TG-2: Super Admin - List and Delete

### PR DESCRIPTION
**Pull request by PilarFernandezC**
- Shows list of admins
- Delete functionality with warning and success message
- Edit and Add buttons that redirects you to add form

### CODE REVIEW

File: src/Components/SuperAdmins/Modal/modal.module.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.delete {
+    margin: 5px;
+    width: 80px;
+    height: 40px;
+    border-radius: 5px;
+    border-color: #F13312;
+    background-color: #F13312;
+    color: white;
```
Comment (martumoser) [inline src/Components/SuperAdmins/Modal/modal.module.css:38]: Please keep consistency between the use of colors on hex or with the color name. We suggest to use color hex

File: src/Components/Layout/index.js [hunk 2]
```diff
@@ -30,6 +31,9 @@ function Layout() {
     case '/tasks':
       currentScreen = <Tasks />;
       break;
+    case '/super-admins/form':
+      currentScreen = <SuperAdminForm />;
+      break;
```
Comment (martumoser) [inline src/Components/Layout/index.js:36]: Please move this case to lines 21, above the /super-admins case

File: src/Components/SuperAdmins/List/List.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ListItem from '../ListItem/ListItem';
+import styles from './list.module.css';
+
+const List = ({ admin, handleDelete }) => {
```
Comment (martumoser) [inline src/Components/SuperAdmins/List/List.jsx:5]: in this case admin is plural (admins) because it is the array with all admins

**Review by martumoser — CHANGES_REQUESTED** [review]

File: src/Components/SuperAdmins/Modal/modal.module.css (hunk 1 above)
Comment (PilarFernandezC) [reply src/Components/SuperAdmins/Modal/modal.module.css]: done

File: src/Components/Layout/index.js (hunk 2 above)
Comment (PilarFernandezC) [reply src/Components/Layout/index.js]: done

File: src/Components/SuperAdmins/List/List.jsx (hunk 3 above)
Comment (PilarFernandezC) [reply src/Components/SuperAdmins/List/List.jsx]: done

**Review by martumoser — APPROVED** [review]

**Review by DamianPalavecino — APPROVED** [review]

**Pull request merged**